argoverse/data_loading/argoverse_stereo_loader.py

```diff
@@ -0,0 +1,432 @@
+# <Copyright 2021, Argo AI, LLC. Released under the MIT license.>
+
+import glob
+import logging
+import os
+from typing import Dict, Iterator, List, Optional, Union
+
+import cv2
+import numpy as np
+
+from argoverse.utils.calibration import Calibration, load_calib
+from argoverse.utils.camera_stats import STEREO_CAMERA_LIST
+
+logger = logging.getLogger(__name__)
+
+STEREO_FRONT_LEFT_RECT = STEREO_CAMERA_LIST[2]
+STEREO_FRONT_RIGHT_RECT = STEREO_CAMERA_LIST[3]
+
+
+class ArgoverseStereoLoader:
+    def __init__(self, root_dir: str, split_name: str) -> None:
+        # initialize class member
+        self.STEREO_CAMERA_LIST = STEREO_CAMERA_LIST
+        self._log_list: Optional[List[str]] = None
+        self._image_list: Optional[Dict[str, Dict[str, List[str]]]] = None
+        self._disparity_list: Optional[Dict[str, Dict[str, List[str]]]] = None
+        self._image_timestamp_list: Optional[Dict[str, Dict[str, List[int]]]] = None
+        self._timestamp_image_dict: Optional[Dict[str, Dict[str, Dict[int, str]]]] = None
+        self._calib: Optional[Dict[str, Dict[str, Calibration]]] = None  # { log_name: { camera_name: Calibration } }
```

Comment:
is `# { log_name: { camera_name: Calibration } }` this commented out code intentionally left in?

Reply:
Removed it.